sceneformux/ux/src/main/java/com/google/ar/sceneform/ux/ScaleController.java

```diff
@@ -140,7 +144,11 @@ public void onContinueTransformation(PinchGesture gesture) {
   }
 
   @Override
-  public void onEndTransformation(PinchGesture gesture) {}
+  public void onEndTransformation(PinchGesture gesture) {
+    if (null != listener) {
```

Comment:
Personally I prefer the order that reads in English as "if listened isn't null":

```suggestion
    if (listener != null) {
```